subsys/bluetooth/host/gatt.c

```diff
@@ -455,6 +455,60 @@ static ssize_t sc_ccc_cfg_write(struct bt_conn *conn,
 	return sizeof(value);
 }
 
+#if defined(CONFIG_BT_SETTINGS) && defined(CONFIG_BT_SMP)
+static int bt_gatt_store_cf(struct bt_conn *conn);
```

Comment:
Is there really no way to avoid this, e.g. by moving stuff up or down in the c-file? Note: I'm not asking you to necessarily do something, if it's too invasive, just that you checked what it would mean if one would try to avoid it.

Reply:
Found a way.